packages/bare_encoding/bare_encoding.0.1.1/opam

```diff
@@ -0,0 +1,36 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "BARE encoding, see https://baremessages.org/"
+maintainer: ["simon cruanes"]
+authors: ["simon cruanes"]
+license: "MIT"
+tags: ["encoding" "binary" "bare"]
+homepage: "https://github.com/c-cube/bare-ocaml"
+bug-reports: "https://github.com/c-cube/bare-ocaml/issues"
+depends: [
+  "dune" {>= "2.8" & >= "2.5"}
```

Comment:
```suggestion
  "dune" {>= "2.8"}
```
Could you remove the redundant dune dependency from the dune-project file? See https://github.com/ocaml/dune/issues/3700